course_catalog/serializers.py

```diff
@@ -45,15 +45,15 @@ def to_representation(self, instance):
         # Pass context on to the serializers so that they have access to the user
         context = self.context
         if isinstance(instance, Bootcamp):
-            serializer = SimpleBootcampSerializer(instance, context=context)
+            serializer = BootcampSerializer(instance, context=context)
         elif isinstance(instance, Course):
-            serializer = SimpleCourseSerializer(instance, context=context)
+            serializer = CourseSerializer(instance, context=context)
```

Comment:
With runs now included here, getting quite a few N+1 queries.  Might be worth trying to eliminate them.
```
web_1      | [2020-01-09 19:56:43] ERROR 28 [nplusone] notifiers.py:40 - [035bea53deb6] - Potential n+1 query detected on `LearningResourceRun.learningresourceofferor_set`
web_1      | [2020-01-09 19:56:43] ERROR 28 [nplusone] notifiers.py:40 - [035bea53deb6] - Potential n+1 query detected on `LearningResourceRun.instructors`
web_1      | [2020-01-09 19:56:43] ERROR 28 [nplusone] notifiers.py:40 - [035bea53deb6] - Potential n+1 query detected on `LearningResourceRun.instructors`
web_1      | [2020-01-09 19:56:43] ERROR 28 [nplusone] notifiers.py:40 - [035bea53deb6] - Potential n+1 query detected on `LearningResourceRun.courseprice_set`
web_1      | [2020-01-09 19:56:43] ERROR 28 [nplusone] notifiers.py:40 - [035bea53deb6] - Potential n+1 query detected on `LearningResourceRun.courseprice_set`
```